platform/ios/src/MGLLocationManagerUtility.h

```diff
@@ -0,0 +1,25 @@
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+
+@protocol MGLLocationManagerUtilityDelegate;
+
+@interface MGLLocationManagerUtility : NSObject <CLLocationManagerDelegate>
```

Comment:
Maybe call this MGLLocationManager or MGLTelemetryLocationManager, since it functions like a replacement for CLLocationManager rather than a complement to it.